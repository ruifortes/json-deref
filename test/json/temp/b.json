 {
   "a": {"$ref": "#b/b/a"},
   "b": {
     "a": 1,
     "b": {
       "a": 2,
       "b": {"$ref": "#c"}
     }
   },
   "c": 3
 }
