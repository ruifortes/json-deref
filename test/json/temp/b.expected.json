 {
   "a": 2,
   "b": {
     "a": 1,
     "b": {
       "a": 2,
       "b": 3
     }
   },
   "c": 3
 }
